Cargo.toml

```diff
@@ -22,6 +22,12 @@ libc = "0.2.58"
 log = "0.4.7"
 pretty_env_logger = "0.3.0"
 once_cell = "0.2.4"
+nodrop = "0.1.14"
```

Comment:
from the crate docs of nodrop

> Deprecated: Use ManuallyDrop or MaybeUninit instead!

Reply:
https://docs.rs/nodrop/0.1.14/nodrop/

Reply:
https://doc.rust-lang.org/stable/std/mem/struct.ManuallyDrop.html

I guess I missed that the first time around

Reply:
fixed